services/viva-ms/helpers/dynamoDb.js

```diff
@@ -0,0 +1,40 @@
+import * as dynamoDb from '../../../libs/dynamoDb';
+import config from '../../../config';
+import { CASE_HTML_GENERATED } from '../../../libs/constants';
+
+export async function getClosedUserCases(primaryKey) {
+  const dynamoDbQueryCasesParams = {
+    TableName: config.cases.tableName,
+    KeyConditionExpression: 'PK = :pk and begins_with(SK, :sk)',
+    FilterExpression: 'begins_with(#status.#type, :statusTypeClosed)',
+    ExpressionAttributeNames: {
+      '#status': 'status',
+      '#type': 'type',
+    },
+    ExpressionAttributeValues: {
+      ':pk': primaryKey,
+      ':sk': 'CASE#',
+      ':statusTypeClosed': 'closed',
+    },
+  };
+  return dynamoDb.call('query', dynamoDbQueryCasesParams);
+}
+
+export async function updateVivaCaseState(caseItem) {
```

Comment:
No need to specify `async` because you don't have any `await` expression in the function body :)

Reply:
Done